tests/test_ban_tk_connect.py

```diff
@@ -29,13 +31,13 @@ def test_ban_excluded_weapon(*args):
         "version": 1,
         "timestamp_ms": 1612695641000,
         "action": "TEAM KILL",
-        "player_name_1": "[ARC] DYDSO ★ツ",
+        "player_name_1": "[ARC] DYDSO",
         "player_id_1": 76561198091327692,
         "player_name_2": "Francky Mc Fly",
-        "player_id_1": 76561198091327692,
+        "player_id_2": 76561198091327692,
         "weapon": "None",
-        "raw": "[646 ms (1612695641)] TEAM KILL: [ARC] DYDSO ★ツ(Axis/76561198091327692) -> Francky Mc Fly(Axis/76561198133214514) with None",
```

Comment:
Any reasons that the special characters were removed? Wasn't it to test the handling of special characters?

Reply:
Hehe, good catch, dammit, wasn't meant to be committed. I'm running on Windows and for whatever reason python uses some windows charmap for strings, and this one does not support these unicode characters. As I wanted to have the tests run, I removed them, as they are not needed for the test itself 👍

Reply:
Do you want to revert that commit to make sure that characters causes no issues?

Reply:
If you can't, I can check that.

Reply:
done :)